internal/vendor/wiz.go

```diff
@@ -230,9 +230,9 @@ type SAMLGroupMappingCreateInput struct {
 type SAMLIdentityProvider struct {
 	AllowManualRoleOverride  *bool               `json:"allowManualRoleOverride"`
 	Certificate              string              `json:"certificate"`
-	Domains                  []string            `json:"domains"`
-	EntityID                 string              `json:"entityID,omitempty"`
-	GroupMapping             []*SAMLGroupMapping `json:"groupMapping,omitempty"`
+	Domains                  []*string           `json:"domains"`
+	IssuerURL                string              `json:"issuerURL"`
+	GroupMapping             []*SAMLGroupMapping `json:"groupMapping"`
```

Comment:
The golang struct should mirror the GraphQL schema (omitempty).  issuerURL and groupMapping allow empty values.

```
type SAMLIdentityProvider {
  allowManualRoleOverride: Boolean!
  certificate: String!
  domains: [String!]
  groupMapping: [SAMLGroupMapping!]
  id: ID!
  issuerURL: String
  loginURL: String!
  logoutURL: String!
  mergeGroupsMappingByRole: Boolean!
  name: String!
  useProviderManagedRoles: Boolean!
}
```